ansible/roles/texlive/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+- name: texlive | prereq
+  yum: name={{ item }} state=latest
+  with_items:
+    - perl-Tk
+    - perl-Digest-MD5
+
```

Comment:
What might work is to ensure the tmp dir doesn't already exist- so if there's a failed installer download lying around the intaller will be re-downloaded.